hooks/read-package-hook/src/createVersionsOverrider.ts

```diff
@@ -20,9 +20,12 @@ export function createVersionsOverrider (
           const pkgPath = override.newPref.substring(5)
           linkTarget = path.isAbsolute(pkgPath) ? pkgPath : path.join(rootDir, pkgPath)
```

Comment:
what about this one? Should probably follow the same rules. Let's refactor both

Reply:
Right now, `link:` converts all absolute path to relative path, but `file:` preserves absolute path. The change you requested would alter one of them. Do you still want to change? If you do, do you prefer both converts absolute path to relative or both preserves absolute path?

Reply:
Isn't this change making file preserve related? The new way `file:` works should be fine for `link:` too.

Reply:
I have made `link:` behaves like `file:` (preserve absolute path)